package.json

```diff
@@ -93,21 +93,21 @@
     "style-loader": "^1.2.1",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.11.0"
+    "webpack-dev-server": "^3.11.0",
+    "@emotion/react": "^11.4.1",
```

Comment:
For development, we need to have @emotion/react installed and make sure it's in a version recent enough for `@mui/material` and `tss-react`.